packages/api/src/pins.js

```diff
@@ -350,6 +350,13 @@ export async function pinDelete (request, env, ctx) {
     )
   }
 
+  if (!(/^\d+$/.test(request.params.requestId))) {
```

Comment:
Can we remove this from this PR? It'll probably create conflicts with the DELETE one

Reply:
Ok have removed.